src/ConnectedMode/ConnectionInfo.cs

```diff
@@ -28,5 +28,5 @@ public enum ServerType
         SonarCloud
     }
 
-    public record Connection(string id, ServerType serverType);
+    public record Connection(string id, ServerType serverType, bool hasSmartNotifications);
```

Comment:
I am not sure about the `has`. SLCore is using `bool RetrieveSmartNotifications`, and on our side, it is currently `bool DisableNotifications`.

Reply:
Morning! Regarding your feedback bellow. You mentioned that on our side is "DisableNotifications", but I couldn't find this string in the whole solution. I named the property hasSmartNotifications, because in our mockup it is called "Smart Notifications". I think it would be nice if the property in the view model reflects the property in the UI. But what would be a better name "SmartNotificationsEnable" or just "SmartNotifications"? The problem with just "SmartNotifications" is that it is not speaking that is a bool value.

Reply:
I renamed it RetrieveSmartNotifications.

Reply:
The `DisableNotifications` is in the `ServerConnectionConfiguration` class. It would be better to check that with Giorgii. I believe we still receive notifications, but we discard them or do not show them.